### ISSUE TITLE
Sync time over Bluetooth 

### ISSUE THREAD

**Issue by ananthb** [mooltipass/minible#381]
### Missing feature

The mooltipass device should sync it's clock over Bluetooth in addition to USB.

### Justification

Tell us why you would like to see this feature added.

TOTPs are unusable from my phone only.
I need to have a PC and a phone to make this work.

### Workarounds

Are there any workarounds you currently have in place because the feature is missing?

I keep my device plugged in to a laptop near me if I know I'm going to need a TOTP.

**Comment by limpkin** [comment]
oh that would be a neat feature for our android-companion indeed :)

**Comment by My1** [comment]
honestly this only makes sense

**Comment by wkennington** [comment]
This is not supported in thefirmware? Or this is just an app thing?

**Comment by KDederichs** [comment]
@limpkin I'm looking into building that into the iOS App and got two questions:
1) What timezone? I'd assume UTC?
2) Does the device need to be unlocked for this? (If so why, I'd request that changed :D)

**Comment by My1** [comment]
I'd guess UTC too as the MP doesnt display the local time and the primary point for the time is TOTP which works off of unixtime

**Comment by limpkin** [comment]
1. UTC indeed, @deXol can yo uconfirm?
2. it does not :)

**Comment by KDederichs** [comment]
Added it to testflight for the iOS app, so if you're iOS crowd feel free to check it out :)

**Comment by deXol** [comment]
> 1. UTC indeed, @deXol can you confirm?
> 
>     2. it does not :)

Yes, it is UTC: https://github.com/mooltipass/moolticute/blob/master/src/MPDevice.cpp#L3901

**Comment by My1** [comment]
cool. next step: the android app.

**Comment by limpkin** [comment]
tracked here: https://github.com/mathfactory/mooltifill-android/issues/3
